### PR TITLE
Layer styling support.

### PR DESCRIPTION
**Pull request by jmalczyk**
Adds a temporary button to switch layers to gray color, to be repaced by a style/color picker widget.

### CODE REVIEW

File: app/js/mol.map.layers.js [hunk 1]
```diff
@@ -205,6 +201,7 @@ mol.modules.map.layers = function(mol) {
                         self.bus.fireEvent(new mol.bus.Event('show-layer-display-toggle'));
 
                         // Set initial opacity based on layer type.
+                        //TODO, pull this from the types metadata table instead
```
Comment (eightysteele) [inline app/js/mol.map.layers.js]: Lets submit an issue for this, and link to it from this comment.

File: app/js/mol.map.search.js [hunk 2]
```diff
@@ -13,24 +13,24 @@ mol.modules.map.search = function(mol) {
...
-                    'FROM layer_metadata l ' +
-                    'LEFT JOIN types t ON ' +
-                    '       l.type = t.type ' +
-                    'LEFT JOIN providers p ON ' +
-                    '       l.provider = p.provider ' +
-                    'LEFT JOIN taxonomy n ON ' +
-                    '       l.scientificname = n.scientificname ' +
-                    'WHERE ' +
-                    "  l.scientificname~*'\\m{0}' OR n.common_names_eng~*'\\m{0}'";
+                'SELECT DISTINCT l.scientificname as name,'+
```
Comment (eightysteele) [inline app/js/mol.map.search.js:16]: Hard to tell... did the query change here or no?

**Comment by eightysteele** [comment]
A few inline nits, otherwise LGTM. #shipit

File: app/js/mol.map.search.js (hunk 2 above)
Comment (jmalczyk) [inline app/js/mol.map.search.js:16]: No, it didn't. Just some spacing issues. This query and the same in mol.map.boot.js will be getting replaced by a postgresql function.

Comment (eightysteele) [inline app/js/mol.map.search.js:16]: Cool.

**Pull request merged**